examples/NewDscResourceScript/resource.ps1.txt

```diff
@@ -1,57 +1,104 @@
-# The Get-<%=$PLASTER_PARAM_TargetResourceName%> function fetches the status of the <%=$PLASTER_PARAM_TargetResourceName%> resource instance specified
+# The Get-TargetResource function fetches the status of the <%=$PLASTER_PARAM_TargetResourceName%> resource instance specified
 # in the parameters for the target machine.
-function Get-<%=$PLASTER_PARAM_TargetResourceName%> {
+function Get-TargetResource {
     [OutputType([Hashtable])]
     param (
 <%
     if ($PLASTER_PARAM_Ensure -eq 'Yes') {
 "        # Ensure the presence/absene of the resource."
+"        [Parameter(Mandatory)]"
 "        [ValidateSet('Present','Absent')]"
-"        [string]"
-"        `$Ensure = 'Present'"
+         if ($PLASTER_PARAM_IsSingleInstance -eq 'Yes') {
+"        [string] `$Ensure,"
+""
+         } else {
```

Comment:
Could you tweak this?  Nowhere else in this code base do we use a cuddled else.  Please move the `else {` to a new line.